app/forms/LrnValidator.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import connectors.CustomsDeclareExportsConnector
+import models.declaration.submissions.{Action, Submission}
+import uk.gov.hmrc.http.HeaderCarrier
+
+import java.time.ZonedDateTime
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class LrnValidator @Inject()(customsDeclareExportsConnector: CustomsDeclareExportsConnector) {
+
+  def hasBeenSubmittedInThePast48Hours(lrn: Lrn)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Boolean] = {
+    val now = ZonedDateTime.now(Action.defaultDateTimeZone)
```

Comment:
Shouldn't this statement, to be precise, be moved inside method `isSubmissionOlderThan48Hours` ?

Reply:
This way all Submissions are compared to the same moment in time. If it was as you're suggesting, then every Submission would be compared against a slightly different moment.
The difference is small, but this how I felt would be more appropriate.

Reply:
You're right. I forgot the requirement... search between all submissions with that LRN if one of them is older then 48h from now